Fix Errors

hannadiamond opened the pull request:
Errors fixed:
1. part_steps not an int so was unable to run for loops
2. opening pickle file was not read in properly so was unable to load pickle file
3. 'sort(item_list[file_dir][file_root]) was unable to compare object CarProp
4. cv2.rectangle had floats for coords
5. coords for cv2 rectangle were out of bounds in the picture

@mundhenk

hannadiamond committed:
Fix Errors

Errors fixed:
1. part_steps not an int so was unable to run for loops
2. opening pickle file was not read in properly so was unable to load pickle file
3. 'sort(item_list[file_dir][file_root]) was unable to compare object CarProp
4. cv2.rectangle had floats for coords
5. coords for cv2 rectangle were out of bounds in the picture

hdiggins commented:
> Errors fixed:
> 
>     1. part_steps not an int so was unable to run for loops
> 
>     2. opening pickle file was not read in properly so was unable to load pickle file
> 
>     3. 'sort(item_list[file_dir][file_root]) was unable to compare object CarProp
> 
>     4. cv2.rectangle had floats for coords
> 
>     5. coords for cv2 rectangle were out of bounds in the picture
@mundhenk

hdiggins commented:
Not sure if you will see this @mundhenk, but hopefully this will ping you

hannadiamond committed:
Update Color Labels

Add comments for color and changed last color from white to purple so it would be more visible

mundhenk commented:
Hi, I was just popping in. I have a project we're using COWC on again. Let me take a look at your fixes. The code is probably not that great. I was more or less starting out with Python at the time. I'm really afraid to look at it again.
